src/MovieSnapShot/MovieSnapShot.js

```diff
@@ -27,12 +27,14 @@ class MovieSnapShot extends Component {
       const cleanedTrailer = trailerCleaner(trailer);
       this.setState({clickedMovieId: id, singleMovie: cleanedSingleMovie, trailer: cleanedTrailer[0]})
     })
-    .catch(error => this.setState({error: error.message}) )
+    .catch(error => this.setState({clickedMovieId: id, error: error.message}) )
   }
 
   render () {
-   if(!this.state.clickedMovieId) {
```

Comment:
We will need to reach out to figure our the best way to test both loading and error. It seams to be working on manual testing but can only force one at a time.